src/indieweb_utils/webmentions/validate.py

```diff
@@ -0,0 +1,159 @@
+from urllib import parse as url_parse
+from typing import List
+
+from bs4 import BeautifulSoup
+import requests
+
+from ..utils.urls import canonicalize_url
+
+
+class WebmentionValidationError(Exception):
+    def __init__(self, message):
```

Comment:
I don't think we need to pass in message, do we? If we do, since it's repeated code, we should make a IndiewebUtilsException in `src/indieweb_utils/exceptions.py` and inherit from that.

Reply:
I have replaced the __init__ constructor with a pass statement and will include this change in my next commit. After a bit of testing, it seems the __init__ code is not necessary to raise an exception that includes a custom error message.